CHANGELOG.md

```diff
@@ -6,12 +6,19 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
-- Move Settings into channel on the homescreen (#689)
 
 ### Changed
 
 ### Fixed
 
+
+## [3.8] 2019-04-30
```

Comment:
Should this be the 24th?

Reply:
We plan to actually release next Tuesday, so generally I push it a few days out.